pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -219,6 +249,14 @@ def submissions(self):
                                        creation_time=self.object.commented_on)
                               .order_by("id"))
 
+    @cached_property
+    def comments(self):
+        return dict(
+            get_model().objects
+                       .filter(suggestions__unit=self.object)
+                       .values_list("suggestions__id", "comment")
+        )
```

Comment:
yep, after looking more at GenericRelation i agree it would be better, pita about Postgres